solidity/scripts/xerc20-deployment-config.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "Test",
     "symbol": "TST",
+    "decimals": 18,
     "chainDetails": [
         {
             "rpcEnvName": "ETHEREUM_GOERLI_RPC",
```

Comment:
Maybe is just naming but, it isnt SEPOLIA here?

Reply:
that's probably there from before Goerli was deprecated